packages/metronome/buildinfo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "requires": ["java", "exhibitor"],
+  "requires": ["java", "exhibitor", "check-time"],
```

Comment:
We use requires for compile time dependencies nowadays only, no need to add check-time here or to minuteman.